apps/cash-register/public/script.js

```diff
@@ -62,11 +62,12 @@ const checkCashRegister = () => {
     if (changeDue > denominations[i] && changeDue > 0) {
       let count = 0;
       let total = reversedCid[i][1];
-      while (total > 0 && changeDue >= denominations[i]) {
+      while (total - denominations[i] >= 0 && changeDue >= denominations[i]) {
         total -= denominations[i];
         changeDue = parseFloat((changeDue -= denominations[i]).toFixed(2));
         count++;
       }
+      
       if (count > 0) {
         result.change.push([reversedCid[i][0], count * denominations[i]]);
```

Comment:
```suggestion
        result.change.push([reversedCid[i][0], (count * denominations[i]).toFixed(2)]);
```
Rounding for status didn't survive merge conflict :slightly_smiling_face: